### ISSUE TITLE
Charts: Choosing the ChartThemeColor.multi option for a pf chart displays colors in the same family 

### ISSUE THREAD

**Issue by jcaianirh** [patternfly/patternfly-react#5591]
**Describe the issue. What is the expected and unexpected behavior?**
Choosing the ChartThemeColor.multi option for a pf chart displays colors in the same family 
See image:

![image](https://user-images.githubusercontent.com/35978579/112679873-0f904c00-8e43-11eb-9bbc-05bd8cdfede5.png)

Could we get a more diverse set of colors when a multi color theme is requested?

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug
**What is your product and what release version are you targeting?**
OpenShift V4.8

**Comment by nicolethoen** [comment]
@dlabrecq Could you take a look at this?

**Comment by dlabrecq** [comment]
The color pallet below is an example that shows all the colors provided by the multi-color theme. However, the chart above has only used the first 2-3 colors from the pallet. 

Note that `ChartThemeColor.multi` is an ordered theme, which uses colors that have been carefully ordered to provide the best contrast. There is also a multi, unordered theme you can try. 

You can provide your own custom colors.

![image](https://user-images.githubusercontent.com/17481322/112739950-b3aeeb80-8f46-11eb-81af-8a08adcb941d.png)

From https://www.patternfly.org/v4/guidelines/colors-for-charts#chart-with-nested-grouped-variables

**Comment by dlabrecq** [comment]
The color pallet below is an example that shows all the colors provided by the multi-color theme. However, each chart above only uses the first 2-3 colors from the pallet. Some of those charts only have one data series, so you only see the first color in the pallet.

Note that `ChartThemeColor.multi` is an ordered theme. This uses colors that have been carefully ordered to provide the best, accessible contrast. 

Multi-ordered theme
https://www.patternfly.org/v4/guidelines/colors-for-charts/#use-cases

![image](https://user-images.githubusercontent.com/17481322/112739950-b3aeeb80-8f46-11eb-81af-8a08adcb941d.png)

Note that there is also a multi, unordered theme you can try, which has a different color pallet.
http://patternfly-react.surge.sh/charts/themed-charts#multi-color-unordered

You can also provide your own custom colors.
See http://patternfly-react.surge.sh/charts/themed-charts#custom-colors

**Comment by dlabrecq** [comment]
These charts deal with "data that includes six or less variables", and these are the recommended base colors for that use case.

Note that `ChartThemeColor.multi` is an ordered theme. It includes colors that have been carefully ordered based on recommended use cases and provide the best, accessible contrast.

The example below shows all the colors provided by the multi-color theme. However, each chart above only uses the first 2-3 colors from the pallet.

Multi-ordered theme
https://www.patternfly.org/v4/guidelines/colors-for-charts/#use-cases

![image](https://user-images.githubusercontent.com/17481322/112739950-b3aeeb80-8f46-11eb-81af-8a08adcb941d.png)

Note that there is also a multi, unordered theme you can try, which has a different color pallet.
http://patternfly-react.surge.sh/charts/themed-charts#multi-color-unordered

Finally, you can also provide your own custom colors.
See http://patternfly-react.surge.sh/charts/themed-charts#custom-colors

**Comment by jcaianirh** [comment]
@dlabrecq yeah so you can see the first 3 colors in multi are all blue greens. It looks like the multiUnordered may be the way to go to achieve a truly variable color pallet. What does the term order refer to here anyway?

**Comment by jcaianirh** [comment]
Thanks @dlabrecq !!
